### PR TITLE
AUTHN-2322: Secure checkout on auth cancellation we reset & authentication release 3.1.0

### PR DESCRIPTION
**Pull request by robertdalmeida**
# Summary
- [x] Authentication SDK version 3.1.0 release
- When user cancels authentication twice in a row we reset the credentials. 
- Update translations from smartling

# Release notes
<new>
Updated Authentication SDK to 3.1.0 with updated configuration for secure checkout passkeys.
</new>

# Ticket

<ticket>
AUTHN-2322
</ticket>

### CODE REVIEW

**Review by atmamont — COMMENTED** [review]
I would always split dependency update from any logic changes, would it be a hassle to split this into two PRs?

**Comment by robertdalmeida** [comment]
> I would always split dependency update from any logic changes, would it be a hassle to split this into two PRs?

@atmamont https://github.com/Adyen/adyen-ios/pull/1894
There was the bots MR that was for the dependency update, it missed a few things that we fixed in this MR. (I don't know how to fix the bot though). Let me know how you want to proceed.

**Comment by atmamont** [comment]
> > I would always split dependency update from any logic changes, would it be a hassle to split this into two PRs?
> 
> @atmamont #1894 There was the bots MR that was for the dependency update, it missed a few things that we fixed in this MR. (I don't know how to fix the bot though). Let me know how you want to proceed.

Totally forgot that, Robert. Sorry, then let's not make this big.

**Review by atmamont — DISMISSED** [review]

**Review by goergisn — DISMISSED** [review]

File: AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift [hunk 1]
```diff
@@ -372,7 +376,8 @@ internal typealias VoidHandler = () -> Void
             return AdyenAuthentication.AuthenticationService(
                 passKeyConfiguration: .init(
                     relyingPartyIdentifier: delegatedAuthenticationConfiguration.relyingPartyIdentifier,
-                    displayName: cardNumber ?? Bundle.main.displayName
+                    displayName: cardNumber ?? Bundle.main.displayName,
+                    consecutiveApprovalCancellationsLimit: 2
```
Comment (erenbesel) [inline AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift]: this number 2 is used a few times. if they refer to the same number, could you put it on a constant?

Comment (robertdalmeida) [reply AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift]: Sure. Done.

**Review by erenbesel — DISMISSED** [review]

**Review by goergisn — APPROVED** [review]

**Review by erenbesel — APPROVED** [review]

**Pull request merged**